modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -462,6 +462,26 @@ protected String getPermissionCheckerResourceName(Object id)
 			_objectDefinition.getObjectDefinitionId();
 	}
 
+	@Override
+	protected void preparePatch(
+		ObjectEntry objectEntry, ObjectEntry existingObjectEntry) {
+
+		if (objectEntry.getProperties() != null) {
+			existingObjectEntry.setProperties(
+				() -> {
+					ObjectEntry getObjectEntry = getObjectEntry(
+						existingObjectEntry.getId());
```

Comment:
@marcelabc I'm just not sure if this name `getObjectEntry` is ok.. but the fix makes sense to me.

Reply:
Same as [DataRecordResourceImpl](https://github.com/carolmariaabb/liferay-portal/blob/ba477766b71bbcbc9d802c33473a31a5baeae56d/modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataRecordResourceImpl.java#L377-L394)